beacon-chain/core/electra/transition.go

```diff
@@ -27,6 +27,9 @@ var (
 	ProcessSyncCommitteeUpdates          = altair.ProcessSyncCommitteeUpdates
 	AttestationsDelta                    = altair.AttestationsDelta
 	ProcessSyncAggregate                 = altair.ProcessSyncAggregate
+	// These need to be rewritten for electra.
+	ProcessDeposits                      = altair.ProcessDeposits
+	ProcessAttestationsNoVerifySignature = altair.ProcessAttestationsNoVerifySignature
```

Comment:
If these are not related to EIP7002, we probably shouldn't have them, even as placeholders it's a little confusing to the reader. Instead, we could open issues to track them if really needed, which is cleaner....

Reply:
will remove